courses/tasks.py

```diff
@@ -0,0 +1,58 @@
+"""
+Tasks for the courses app
+"""
+import logging
+from mitxpro.celery import app
+from mitxpro.utils import now_in_utc
+from courses.models import CourseRun, CourseRunCertificate
+from courses.utils import ensure_course_run_grade, process_course_run_grade_certificate
+from courseware.api import get_edx_grades_with_users
+
+log = logging.getLogger(__name__)
+
+
+@app.task
+def generate_course_certificates():
+    """
+    Task to generate certificates for courses.
+    """
+    log.info("Task generate_course_certificates: Executed")
```

Comment:
Wouldn't it be more accurate to log "Started" here? "Executed" (past tense) implies the task has finished.